templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -0,0 +1,46 @@
+{#
+/**
+ * @file
+ * Default theme implementation for rendering corporate blocks footer.
+ *
+ * Available variables:
+ * - corporate_footer: Array with structure compatible to ECL ec-components-footer component.
+ *   - contact: EU contact data for corporate footer.
+ *   - social_media_title: Title for social media links column.
+ *   - social_media: Social media links for corporate footer.
+ *   - institution_links_title: Title for institution links column.
+ *   - institution_links: EU institutions links for corporate footer.
+ *   - legal_links_title: Title for legal links column.
+ *   - legal_links: Legal links for corporate footer.
+ *
+ * @ingroup themeable
+ */
+#}
```

Comment:
compute _section_2 and _section_3 from the link provided in the ticket (its copy paste)